webpack-lighthouse/server.js

```diff
@@ -4,6 +4,7 @@ const express = require('express');
 const compression = require('compression');
 
 module.exports = async function launchWebpackServer(port = 3000, webpackConfig = {}, webpackDevMiddleWareOptions = {}) {
+	console.log(port);
```

Comment:
Can you please revert changes in this file?

Reply:
whoops 👍